### PR TITLE
Separate OncoTree models to a package

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

File: web/pom.xml [hunk 1]
```diff
@@ -23,6 +23,7 @@
         <spring.version>4.3.8.RELEASE</spring.version>
         <spring.boot.version>1.5.3.RELEASE</spring.boot.version>
         <oncotree.core.version>0.0.1-SNAPSHOT</oncotree.core.version>
```
Comment (zhx828) [inline web/pom.xml]: I noticed the version of the core module is differ from the master. Is it intentional?

Comment (ao508) [reply web/pom.xml]: @zhx828 I doubt that was intentional. Feel free to synchronize them unless we hear otherwise from @sheridancbio or @mandawilson

Comment (zhx828) [reply web/pom.xml]: I updated the version.

Comment (sheridancbio) [reply web/pom.xml]: @zhx828 I think all modules should get the same version number ... Yes, let's synchronize. Good catch.

File: web/pom.xml [hunk 2]
```diff
@@ -21,7 +21,8 @@
         <java.version>1.7</java.version>
         <spring.version>4.3.8.RELEASE</spring.version>
         <spring.boot.version>1.5.3.RELEASE</spring.boot.version>
-        <oncotree.core.version>0.0.1-SNAPSHOT</oncotree.core.version>
+        <oncotree.core.version>1.0.0-SNAPSHOT</oncotree.core.version>
```
Comment (sheridancbio) [inline web/pom.xml]: This looks good, but I think the pom for core also needs to be updated.  Unit tests failed with "Could not find artifact org.mskcc.oncotree:core:jar:1.0.0-SNAPSHOT"

Comment (zhx828) [reply web/pom.xml]: @sheridancbio ah good catch. Just updated.

**Comment by zhx828** [comment]
I would consider doing a squash merge.

**Comment by sheridancbio** [comment]
Yes ... I will squash and merge when the integration tests are complete

**Comment by sheridancbio** [comment]
Thanks @zhx828 !

**Pull request merged**